universal-login-wallet/src/ui/react/Modals/Modal.tsx

```diff
@@ -38,7 +38,7 @@ const Modal = () => {
         </ModalWrapperClosable>
       );
     case 'topUpAccount':
-      return (
+      return config ? (
```

Comment:
Why would config be null?

Reply:
When we click the previous button on topUp modal, we see terms. Accepting them moves us to topUp modal once again (skipping choosing name). Modal component has the function 'useRelayerConfig' which uses 'useAsync'. When we click the previous button and then go to topUp modal, modal component tries to get config but 'useRelayerConfig' is not resolved yet and this is why we have an undefined config for short period of time. 'useAsync' rerenders the page and 'useRelayerConfig' is almost instant so the user is not aware of rerendering.

Another option is to wait until 'config' isn't undefined and render some loader but a short period of time, which would be more like blink than loading.

Reply:
Let's use cache instead